### DISCUSSION TITLE
[CICD] Github Actions와 슬랙 알람 연동하기

### DISCUSSION THREAD

**Discussion by amaran-th** [BDD-CLUB/challenge-every-week-posting#123] [1. 개발 관련 글 모음]
별 거 아닌데 삽질 엄청 했네요ㄱ-
슬랙이 생각보다 많은 기능을 추가할 수 있어서 참 재밌는 것 같습니다~
[게시글 링크](https://amaran-th.github.io/%EC%9D%B8%ED%94%84%EB%9D%BC/[CICD]%20Github%20Actions%EC%99%80%20%EC%8A%AC%EB%9E%99%20%EC%95%8C%EB%9E%8C%20%EC%97%B0%EB%8F%99%ED%95%98%EA%B8%B0/)

**Comment by llddang** [comment]
우왕 깔끔하게 정리 잘했다!!
나중에 사용할 일 있으면 세연이 블로그 참고하겠습니다!!
수고하셨습니다!
